test/iio_scanner.py

```diff
@@ -209,6 +209,15 @@ def find_device(names, uri=None):
             boards = [b]
         else:
             return (False, [])
+    elif config:
+        # Convert config to board class
+        boards = []
+        for dev in config["uri-map"]:
+            hardware_options = config["uri-map"][dev]
+            for hardware in hardware_options.split(","):
+                b = board(hardware, dev)
+                boards.append(b)
```

Comment:
This will cause the test to fail if the hardware and uri combination is not connected or online even without the --error_on_filter option

Reply:
This should be fixed now